src/index.js

```diff
@@ -236,9 +236,12 @@ var MaskedInput = React.createClass({
   },
 
   _keyPressPropName() {
-    return navigator.userAgent.match(/Android/i)
+    if (navigator) {
+      return navigator.userAgent.match(/Android/i)
       ? 'onBeforeInput'
       : 'onKeyPress'
+    }
+    return
```

Comment:
let’s return `onKeyPress` as a fallback. In case someone is doing something terribly weird and deleting or hiding `navigator` from the global scope.